urllib3/util/connection.py

```diff
@@ -77,7 +86,6 @@ def create_connection(address, timeout=socket._GLOBAL_DEFAULT_TIMEOUT,
             err = e
             if sock is not None:
                 sock.close()
-                sock = None
 
     if err is not None:
```

Comment:
Why does it hurt coverage? The `sock = None` does nothing as `sock` isn't used anywhere after that so it's not needed.